src/index.js

```diff
@@ -56,9 +56,11 @@ export function app(state, actions, view, container) {
   }
 
   function resolveNode(node) {
-    return typeof node === "function"
-      ? resolveNode(node(globalState, wiredActions))
-      : node
+    return node === null
+      ? h(node, null)
```

Comment:
`h(null)` returns `{nodeName: null, attributes: {}, children: [], key: undefined}` which is invalid vnode for `patch` function

Reply:
@frenzzy I see, any suggestions? Do you think we should modify it on the patch function?

Reply:
We may add support for vnode type=array inside patch function or implement proper fragments support #530 and return fragment node here

Reply:
When you mean is an invalid vnode for `patch` function, what it'll happen when `nodeName: null`? I'm using it right now with the route and it will just render `<null></null>` on the dom but then is removed. It works perfectly and does not show any error or that it renders `<null></null>` is bad? What sort of errors could happen?

Reply:
`null` should render nothing

Reply:
Following our discussion in this PR, this line should definitely not lead to `h(null)`